book/07-git-tools/sections/credentials.asc

```diff
@@ -190,11 +190,11 @@ username=bob
 password=s3cre7
 ----
 
-Since its name starts with ``git-'', we can use the simple syntax for the configuration value:
+Dahil sa pangalang ito na nagsisimula ng ``git-'', maaari nating gamitin ang simpleng syntax para sa halaga ng pagsasaayos:
```

Comment:
change Dahil sa pangalang ito na to Dahil ang pangalang ito ay

Reply:
done change